cms/templates/emails/user_task_complete_email.txt

```diff
@@ -12,13 +12,16 @@ ${_("Your {task_name} task has completed with the status '{task_status}'. Sign i
 
 % if olx_validation_errors:
 
-${_("Here are some validation errors we found with your course content.")}
+${_("Here are some validations we found with your course content.")}
 
-
-% for error in error_report:
-    ${error}
+${_("Errors:")}
+% for error in olx_validation_errors.get('errors'):
+  * ${error}
```

Comment:
Can we translate the error messages as well?

Reply:
I am not sure about the translation because these messages would be highly technical and variable in terms of arguments so their translation is not possible. For example, there are variables like random filenames which could never be translated. See the example below.

```
_('Unknown User ID: {0}').format(user_id)
```

Here the `user_id` is a variable so there would be 1 translation for each language keeping the variable as is. 

So In our case its not possible.

Reply:
From my understanding the translation ignores the variables and it also ignore anything that does not have an available translation. So in the example above it should translate the "Unknown User"

Reply:
Yes -- but the string `error` would contain strings like 
```

ERROR UnexpectedTag (course/2016_R1.xml): A <wiki> tag was unexpectedly found inside the <course url_name='2016_R1' display_name='Xseries Certs Course 2'> tag
```

its preformatted string and its difficult to have translations for all values.